### PR TITLE
Convert or remove AE-likes serving as roll options

### PR DESCRIPTION
**Pull request by stwlam**

### CODE REVIEW

File: packs/data/feats.db/crane-flutter.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
                 "key": "FlatModifier",
                 "predicate": [
                     "crane-flutter",
-                    "crane-stance"
+                    "self:effect:crane-stance"
```
Comment (simonward) [inline packs/data/feats.db/crane-flutter.json:43]: May not need this, as "crane-flutter" is false if not "self:effect:crane-stance" in the above RollOption.

File: packs/data/feat-effects.db/effect-rage.json [hunk 2]
```diff
@@ -43,11 +43,9 @@
                 "value": "@actor.flags.pf2e.rageTempHP"
             },
             {
-                "key": "ActiveEffectLike",
-                "mode": "override",
-                "path": "flags.pf2e.rollOptions.all.rage",
-                "priority": 10,
-                "value": true
+                "domain": "all",
+                "key": "RollOptions",
```
Comment (TikaelSol) [inline packs/data/feat-effects.db/effect-rage.json:47]:
```suggestion
                "key": "RollOption",
```
Also is this entire RE even still necessary at this point?

Comment (stwlam) [reply packs/data/feat-effects.db/effect-rage.json]: Will people who haven't updated their barbarians lose functionality if it's removed?

File: packs/data/feat-effects.db/stance-crane-stance.json [hunk 3]
```diff
@@ -60,11 +60,9 @@
                 "title": "Crane Stance"
             },
             {
-                "key": "ActiveEffectLike",
-                "mode": "override",
-                "path": "flags.pf2e.rollOptions.all.crane-stance",
-                "priority": 10,
-                "value": true
+                "domain": "all",
```
Comment (simonward) [inline packs/data/feat-effects.db/stance-crane-stance.json:63]: This RollOption may not be needed any longer either.

Comment (stwlam) [reply packs/data/feat-effects.db/stance-crane-stance.json:63]: It is still needed.

File: packs/data/feat-effects.db/effect-rage.json (hunk 2 above)
Comment (TikaelSol) [reply packs/data/feat-effects.db/effect-rage.json]: Possibly? Those would be some old barbarians.

**Pull request merged**